jerry-main/main-unix.c

```diff
@@ -427,6 +430,7 @@ main (int argc,
   jerry_init_flag_t flags = JERRY_INIT_EMPTY;
 
   const char *exec_snapshot_file_names[argc];
+  uint32_t exec_snapshot_file_inidicies[argc];
```

Comment:
typo, just spotted: inidicies -> indices (2 extra 'i's) (likewise throughout the file)